.changeset/fuzzy-kids-grin.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Is this file intentionally "blank"?

Reply:
Yes, it was to pass the changeset check. I don't know if it's required, but @dlarocque said to add an empty one.

Reply:
Sorry, you don't need to do this. Our changeset checker CI workflow is broken so it will error, but it's not a required check. Sorry about that.

Reply:
my bad!